test/_helpers.ts

```diff
@@ -38,22 +39,33 @@ export const time = {
   }
 }
 
-export const getFixedGasSigners = async function(gasLimit:number) {
-  const signers : SignerWithAddress[] = await ethers.getSigners();
+export const getSelloutRepayAmount = async function (
```

Comment:
Do we have simple unit tests for this logic?

Reply:
Now we do